### ISSUE TITLE
Specifying empty params (e.g. clusterOptions in profile) breaks with nf-validation 0.3.1

### ISSUE THREAD

**Issue by pontus** [nextflow-io/nf-validation#84]
When running one of  the pipelines using nf-validation with 0.3.1 and specifying the UPPMAX profile, it fails with an error:

```
pontus@lalage:/tmp$ nextflow run nf-core/atacseq -r 2.1.1 -profile uppmax
N E X T F L O W  ~  version 23.04.2
Launching `https://github.com/nf-core/atacseq` [nostalgic_euler] DSL2 - revision: 415795d3c1 [2.1.1]
ERROR ~ JSONObject["project"] not found.

 -- Check script '/home/pontus/.nextflow/assets/nf-core/atacseq/main.nf' at line: 52 or see '.nextflow.log' file for more details
pontus@lalage:/tmp$ 
```

after some digging, it seems to me this is because the profile sets that up in params, although with a null value.

It looks to me that is enough to have it seen by nf-validation, but when nf-validation tries to report it (https://github.com/nextflow-io/nf-validation/blob/01a3131ab7fb2e8cc9e2c60dc97ac978b89b17fb/plugins/nf-validation/src/main/nextflow/validation/SchemaValidator.groovy#L315) it's not found in `paramsJSON` because that's based on the cleaned version (https://github.com/nextflow-io/nf-validation/blob/01a3131ab7fb2e8cc9e2c60dc97ac978b89b17fb/plugins/nf-validation/src/main/nextflow/validation/SchemaValidator.groovy#L271) which has removed anything evaluating to false.

A workaround (that makes sense) for now is to ignore the affected param, but this seems to be a bug that can be difficult to figure out and would be nice to have fixed.

**Comment by pontus** [comment]
And upon rechecking, this is probably the same as #48.

**Comment by ewels** [comment]
Closing as hopefully solved in #89, which I just merged. Shout if that's not the case, and thanks for reporting @pontus 🙏🏻
